addons/binding/org.openhab.binding.alarm/OSGI-INF/org.openhab.binding.alarm.handler.AlarmThingHandlerFactory.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file because it is generated by DS annotations. Also add a .gitignore to the OSGI-INF directory that ignores `/*.xml` files (see other bindings).